src/app/ubs-admin/components/ubs-admin-tariffs/ubs-admin-tariffs.component.ts

```diff
@@ -1,19 +1,48 @@
-import { Component } from '@angular/core';
+import { Component, OnInit, OnDestroy } from '@angular/core';
 import { MatDialog } from '@angular/material/dialog';
 import { UbsAdminTariffsAddServicePopupComponent } from './ubs-admin-tariffs-add-service-popup/ubs-admin-tariffs-add-service-popup.component';
 import { UbsAdminTariffsDeletePopupComponent } from './ubs-admin-tariffs-delete-popup/ubs-admin-tariffs-delete-popup.component';
+import { TariffsService } from '../../services/tariffs.service';
+import { takeUntil } from 'rxjs/operators';
+import { Bag } from '../../models/tariffs.interface';
+import { OrderService } from '../../../main/component/ubs/services/order.service';
+import { Locations } from '../../../main/component/ubs/models/ubs.interface';
+import { LocalStorageService } from '@global-service/localstorage/local-storage.service';
+import { Subject, ReplaySubject } from 'rxjs';
 
 @Component({
   selector: 'app-ubs-admin-tariffs',
   templateUrl: './ubs-admin-tariffs.component.html',
   styleUrls: ['./ubs-admin-tariffs.component.scss']
 })
-export class UbsAdminTariffsComponent {
+export class UbsAdminTariffsComponent implements OnInit, OnDestroy {
+  minAmountOfBigBags: number;
+  locations: Locations;
+  isLoadBar: boolean;
+  isFetching = false;
+  selectedLocationId;
+  bags: Bag[];
+  private destroyed$: ReplaySubject<any> = new ReplaySubject<any>(1);
+  private destroy: Subject<boolean> = new Subject<boolean>();
+  public currentLanguage: string;
```

Comment:
two types of destroy? i guess one is enough.

Reply:
done